src/components/widget/MasterWidget.js

```diff
@@ -31,23 +31,16 @@ class MasterWidget extends Component {
     data: '',
   };
 
-  /**
-   * @method componentDidMount
-   * @summary ToDo: Describe the method.
-   */
   componentDidMount() {
-    const { data, widgetData } = this.props;
+    const { data, widgetData, clearValue } = this.props;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['clearValue' is missing in props validation (react/prop-types)](https://app.codacy.com/app/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4578381)